packages/E2ETest/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -219,6 +219,16 @@
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" />
+
+  <PropertyGroup>
+    <BundleCommand>
+    cd
+    yarn run bundle
+    </BundleCommand>
+  </PropertyGroup>
+  <Import Project="..\..\..\..\vnext\ReactNativeWindows.props" />
```

Comment:
We should add something to the default template too. --  We'll need C# and c++ versions too.
I think we should have a props file generated by default which provides us a single place for apps to configure the build in various ways (hermes vs V8, bundle vs no bundle etc)

Reply:
We also need a better way of determining the path, since react-native-windows could be installed in various different locations.  -- Shouldn't we need to use the $ReactNativeWindowsPath variable?

Reply:
template change and playground change will be in another PR.
I will create a bug to track your topic: (hermes vs V8, bundle vs no bundle etc). In this PR, I only restricted it to Bundle.props and Bundle.targets. There should be another real ReactNativeWindows.props/targets to includes all the things you mentioned.

Yes, for internal project, we can use ReactNativeWindowsPath. I don't know if we need to make it to template.